infra/terraform/test_fixtures/terraform-google-kubernetes-engine.tf

```diff
@@ -71,6 +71,17 @@ resource "google_project_iam_binding" "ci_kubernetes_engine_kms_access" {
   ]
 }
 
+resource "google_project_iam_binding" "ci_kubernetes_engine_kms_admin_access" {
+  provider = "google.phoogle"
+
+  project = "${google_project_services.ci_kubernetes_engine.project}"
+  role    = "roles/cloudkms.admin"
+
+  members = [
+    "serviceAccount:service-${google_project.ci_kubernetes_engine.number}@container-engine-robot.iam.gserviceaccount.com",
```

Comment:
The error we're trying to solve is caused by insufficient permissions when creating a `google_kms_key_ring` resource. Should this role not be added to the service account created by  `google_service_account.ci_kubernetes_engine`?